cartography/data/jobs/cleanup/gcp_dns_cleanup.json

```diff
@@ -0,0 +1,29 @@
+{
+  "statements": [
+    {
+      "query": "MATCH (n:GCPDNSZone) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
Remember to also add indexes for the new nodes. Oh, and maybe update the README.md with the new GCP datatypes so we can show them off.

Reply:
Very good point, thanks for reminding me about the indexes! Adding now